### PR TITLE
Update for v7

### PR DESCRIPTION
**Pull request by FreddieAkeroyd**
See ISISComputingGroup/IBEX#4599

### CODE REVIEW

File: EPICSDoxyApp/src/dbToDoxygen.cpp [hunk 1]
```diff
@@ -20,7 +20,7 @@
 #include "gpHash.h"
 #include "osiFileName.h"
 
-DBBASE *pdbbase = NULL;
+static DBBASE *myPdbBase = NULL;
```
Comment (Tom-Willemsen) [inline EPICSDoxyApp/src/dbToDoxygen.cpp:23]: Why was this change needed and is it documented anywhere?

Comment (FreddieAkeroyd) [reply EPICSDoxyApp/src/dbToDoxygen.cpp:23]: The previous approach was not strictly correct, but worked. The program is not really an IOC and the above gave a conflict in variable name. I followed https://epics.anl.gov/base/R3-16/1-docs/AppDevGuide/StaticDatabaseAccess.html to create and load a local database

**Review by Tom-Willemsen — APPROVED** [review]
